extensions/helix/package.build.ts

```diff
@@ -274,6 +274,225 @@ export const pkg = (modules: Builder.ParsedModule[]) => ({
             },
           },
         },
+
+      "space": {
+        title: "Space",
+        items: {
+          "f": { text: "Open file picker", command: "workbench.action.quickOpen" },
+          "F": {
+            text: "Open file picker at current working directory",
+            command: "dance.run",
+            args: [
+              {
+                code: [
+                  `
+                  const editor = vscode.window.activeTextEditor;
+                  const fallback = () => vscode.commands.executeCommand(
+                      'workbench.action.quickOpen',
+                    );
+
+                  if (!editor) {
+                    return await fallback();
+                  }
+
+                  const currentFileUri = editor.document.uri;
+                  const currentDirectoryUri = vscode.Uri.joinPath(currentFileUri, '..');
+
+                  const workspaceFolder = vscode.workspace.getWorkspaceFolder(currentFileUri);
+                  if (!workspaceFolder) {
+                    return await fallback()
+                  }
+
+                  const relativeDirectoryPath = vscode.workspace.asRelativePath(
+                      currentDirectoryUri,
+                      false
+                    );
+
+                  const quickOpenPrefix = relativeDirectoryPath.endsWith('/')
+                    ? relativeDirectoryPath
+                    : \`\${relativeDirectoryPath}/\`;
+
+                  await vscode.commands.executeCommand(
+                    'workbench.action.quickOpen',
+                    quickOpenPrefix
+                  );
+
+                  `.split("\n").map(line => line.trim()).join(""),
+                ],
```

Comment:
nit: you can also keep this as an array to be more readable (also removing the opening `[`)

```suggestion
                  `.trim().split("\n"),
```

To remove the indentation, I would do `.replaceAll(/^ {,18}/mg, "")` or something like this to preserve the "internal" indentation.

Reply:
Sorry if that wasn't clear, I meant writing the text multiline using a template literal

```js
code: `
  if (...) {
    ...
  }`.replaceAll(/^ {,2}/mg, "")
```

This way it is a multiline string which is easy to read and edit in `package.build.ts`, and in the generated `package.json` it is an array of lines which is similarly easy to read (but less so than a multiline string).